CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKAxesRenderable.java

```diff
@@ -563,6 +579,42 @@ private int CreateVertexBuffers(MemoryStack stack, CVKSwapChain cvkSwapChain) {
     }
   
     
+    private void UpdatePushConstants(CVKSwapChain cvkSwapChain){
```

Comment:
Maybe comment out (otherwise I might steal your untested code and then blame you when it fails in my class :) ).